src/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/RecipientInfo.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+using Internal.Cryptography;
+
+namespace System.Security.Cryptography.Pkcs
+{
+    public abstract class RecipientInfo
+    {
+        internal RecipientInfo(RecipientInfoType type, RecipientInfoPal pal)
+        {
+            switch (type)
```

Comment:
SubjectIdentifier put the whole switch in #if DEBUG.  I don't have a strong care for one way or the other, but it feels like they should be consistent.